### PR TITLE
GAWB-3023: Isolate pet tests in their own projects

### PR DESCRIPTION
**Pull request by jmthibault79**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
Also verifies that GAWB-2867 is not a problem.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Sam.scala [hunk 1]
```diff
@@ -25,10 +25,8 @@ object Sam extends FireCloudClient with LazyLogging with ScalaFutures{
 
   def petName(userInfo: UserStatusDetails) = ServiceAccountName(s"pet-${userInfo.userSubjectId}")
 
-  def removePet(userInfo: UserStatusDetails): Unit = {
-    Sam.admin.deletePetServiceAccount(userInfo.userSubjectId)(UserPool.chooseAdmin.makeAuthToken())
-    // TODO: why is this necessary?  GAWB-2867
```
Comment (jmthibault79) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Sam.scala:30]: Narrator: it wasn't.

**Review by helgridly — APPROVED** [review]
seems legit

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Sam.scala (hunk 1 above)
Comment (rtitle) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Sam.scala:30]: Please narrate all my PRs

**Review by rtitle — APPROVED** [review]
Lgtm

**Comment by jmthibault79** [comment]
Swatomation-Pipeline result is here: https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-PR-fiab-test/1039/

The one failure involves Auth Domains which are unrelated to my changes.

**Review by gdlex4015 — APPROVED** [review]

**Pull request merged**